<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opensimulator Fernsteuerung</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="consolePane">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>Hier befinden sich alle ihre gespeicherten Verbindungen.</string>
          </property>
          <property name="text">
           <string>Verbindungen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="connList">
          <property name="editTriggers">
           <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Connection">
    <property name="title">
     <string>&amp;Verbindung</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_New"/>
    <addaction name="actionNew_in_group"/>
    <addaction name="action_Edit"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionNew_group"/>
    <addaction name="actionDelete_group"/>
   </widget>
   <widget class="QMenu" name="menu_Server">
    <property name="title">
     <string>&amp;Server</string>
    </property>
    <addaction name="action_Clear_text"/>
    <addaction name="separator"/>
    <addaction name="action_Restart"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Hilfe</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Connection"/>
   <addaction name="menu_Server"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Connect"/>
   <addaction name="action_Disconnect"/>
   <addaction name="separator"/>
   <addaction name="action_New"/>
   <addaction name="action_Edit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="action_Restart"/>
   <addaction name="action_Clear_text"/>
   <addaction name="separator"/>
   <addaction name="action_Preferences"/>
  </widget>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/system-log-out.png</normaloff>:/Icons/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ende</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/network-transmit-receive.png</normaloff>:/Icons/network-transmit-receive.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Verbinden</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/network-offline.png</normaloff>:/Icons/network-offline.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Trennen</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/document-new.png</normaloff>:/Icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Neu...</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/document-properties.png</normaloff>:/Icons/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bearbeiten</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/edit-delete.png</normaloff>:/Icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Löschen</string>
   </property>
  </action>
  <action name="actionNew_group">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/folder.png</normaloff>:/Icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Gruppe hinzufügen ...</string>
   </property>
  </action>
  <action name="actionDelete_group">
   <property name="text">
    <string>Gruppe löschen</string>
   </property>
  </action>
  <action name="action_Restart">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/view-refresh.png</normaloff>:/Icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
  <action name="action_Clear_text">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/edit-clear.png</normaloff>:/Icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Löschen</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/edit-copy.png</normaloff>:/Icons/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Kopieren</string>
   </property>
  </action>
  <action name="actionNew_in_group">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/document-new.png</normaloff>:/Icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Neue Gruppe</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/systemPreferences.png</normaloff>:/Icons/systemPreferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Einstellungen</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
